### PR TITLE
Publish Qt/PyQt update announcement

### PR DESCRIPTION
**Pull request by andfoy**
This PR includes a public announcement about the new Qt version.

See https://github.com/conda-forge/pyqt-feedstock/pull/107#issuecomment-1103414256

PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

### CODE REVIEW

**Comment by beckermr** [comment]
I am happy to merge this. 

One question: Who is planning to submit the migrator to regro/cf-scripts?

**Comment by h-vetinari** [comment]
> Who is planning to submit the migrator to regro/cf-scripts?

I think this will happen soon after the packages are built (because people are motivated for this, from what I can tell). Worst case, if that doesn't happen, I can try to submit the migrator.

File: src/user/announcements.rst [hunk 1]
```diff
@@ -8,6 +8,22 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
...
+    After more than six months, the conda-forge team and contributors have managed
+    to update the Qt5 packages to the latest LTS version, 5.15.2. Major changes include
+    separate packages for QtWebEngine (``qt-webengine``) and the rest of Qt in a new
+    package called ``qt-main``. This allows recipes that do not use any of the
+    WebEngine components to depend only on ``qt-main``, reducing the total size of
+    the downloaded binaries. As a result of this, ``qt`` will be a metapackage that
+    installs both ``qt-main`` and ``qt-webengine`` as dependencies.
+
+    With respect to PyQt, the new packages now are in sync with respect to their
+    corresponding PyPi releases, which means that the ``pyqt`` package will only provide
```
Comment (leofang) [inline src/user/announcements.rst:22]: nit:
```suggestion
    corresponding PyPI releases, which means that the ``pyqt`` package will only provide
```

File: src/user/announcements.rst [hunk 2]
```diff
@@ -8,6 +8,22 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2022
 ----
 
+:2022-04-23: Packages for Qt/PyQt 5.15.2 are now available
+
+    After more than six months, the conda-forge team and contributors have managed
+    to update the Qt5 packages to the latest LTS version, 5.15.2. Major changes include
+    separate packages for QtWebEngine (``qt-webengine``) and the rest of Qt in a new
+    package called ``qt-main``. This allows recipes that do not use any of the
```
Comment (leofang) [inline src/user/announcements.rst:16]: nit:
```suggestion
    separating the package for QtWebEngine (``qt-webengine``) from the rest of Qt (now in a new
    package called ``qt-main``). This allows recipes that do not use any of the
```

**Comment by beckermr** [comment]
Wait I thought the packages were already built?

**Comment by h-vetinari** [comment]
> Wait I thought the packages were already built?

@isuruf [asked](https://github.com/conda-forge/pyqt-feedstock/pull/107#issuecomment-1103012703) for the announcement before merging the pyqt-PR. So now things are a bit in limbo with both PRs waiting for each other. They're both ready though, and - AFAICT - could be merged in any order or contemporaneously.

**Comment by beckermr** [comment]
Ah ok. Lgtm!

**Pull request merged**